### PR TITLE
Pf/pf assignment 3

### PR DESCRIPTION
**Pull request by sharma-mohit-eca**
Question 1:- String operations
Question 2:- Student Marksheet
Question 3:- Area Calculator

### CODE REVIEW

**Review by nehajain9302 — CHANGES_REQUESTED** [review]
1. Write different test methods for every method.
2. for every method add positive, negative and exception test cases.
3. Exception handling in code.
4. Longest word method optimization.

**Review by nehajain9302 — COMMENTED** [review]
- I think we can cover all negative and exception scenarios in single test class, so two test classes can be removed.
- Test method name should clearly explain its functionality
- Try to cover all possible conditions applicable in test class : negative value, boundary value and null value. For example these can be used in string input: /'  , '   ' , 'hjhjgj8989', 'w24$#@#'

File: PF/PFAssignment3/StringOperations/StringOperations.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	}
+
+	/**
+	 * Checks whether Entered String is equal to given String
+	 * 
+	 * @param checkEqualityString
+	 * @return 1 if both strings are equal otherwise 0
+	 * @throws StringException
+	 */
+	public int isEquals(String checkEqualityString) throws StringException {
```
Comment (nehajain9302) [inline PF/PFAssignment3/StringOperations/StringOperations.java:28]: I believe return type of this method should be boolean.
Also there should not be more than 1 or 2 return statements. Maybe you can store return value in a variable.

Comment (sharma-mohit-eca) [reply PF/PFAssignment3/StringOperations/StringOperations.java:28]: Ma'am in the Question it is stated that return 1 if equals else 0.

**Review by nehajain9302 — APPROVED** [review]
